apis/relay/builder/blocks.yaml

```
@@ -47,6 +50,8 @@ post:
       application/octet-stream:
         schema:
           description: "SSZ serialized request. Use content type header to indicate that SSZ data is contained in the request body."
+        encoding:
+          Content-Encoding: gzip
```

following the examples here: https://spec.openapis.org/oas/latest.html#considerations-for-file-uploads

it looks like we want this instead
```suggestion
          contentEncoding: gzip
```

and also declared under content.application/json.schema

---

thanks for helping me! new commit made